backend/src/core/config.ts

```diff
@@ -459,9 +459,7 @@ const config = convict({
 })
 
 // If mailFrom was not set in an env var, set it using the app_name
-const defaultMailFrom = `${config.get(
-  'APP_NAME'
-)} <donotreply@mail.postman.gov.sg>`
+const defaultMailFrom = `${config.get('APP_NAME')} <donotreply@postman.gov.sg>`
```

Comment:
Do we need to change this for `worker` and `serverless` too?

Reply:
Ah yes. Good catch!